src/composeappmanager.cc

```diff
@@ -110,6 +110,12 @@ ComposeAppManager::ComposeAppManager(const PackageConfig& pconfig, const Bootloa
     if (cfg_.compose_bin.filename().compare("docker") == 0) {
       compose_cmd += "compose ";  // "make it `docker compose` command
     }
+    std::string out;
+    if (Utils::shell(compose_cmd, &out, true) != EXIT_SUCCESS) {
```

Comment:
I think we can be more clear by not providing a default in composemanager.h. Then in this file we dynamically decide what the default is in order of preference(golang then python). So sort of:
```
if (raw.count("docker_compose_bin") == 0) {
    if exists("/usr/lib/docker/cli-plugins/docker-compose") {
      we use golang version
   } else if exists("/usr/bin/docker-compose") {
     we use python version
   } else {
     this is really bad
   }
}
```

Reply:
Also, how aklite can know that `docker compose` is installed in `"/usr/lib/docker/cli-plugins/docker-compose"` and the same for `docker-compose`. For example, in my local setup I use different locations.

So, I suggest to not make assumption about actual location of `docker` and `docker-compose`.